bundles/core/pom.xml

```diff
@@ -150,6 +150,7 @@
                                         <!-- private enum; cannot test valueOf / values -->
                                         <exclude>**/SocialMediaHelperImpl$WebsiteMetadata$Type.class</exclude>
                                         <exclude>**/AdaptiveImageServlet$Source.class</exclude>
+                                        <exclude>**/HierarchyComponentContextWrapper.class</exclude>
```

Comment:
Why are we excluding `HierarchyComponentContextWrapper` from the coverage?

Reply:
testing simple getters and setters did not seem like a valid test. It was about mocking all the relevant getters and setters in `ComponentContext` and `Page` and asserting that the  `HierarchyComponentContextWrapper` is returning the mocked values.

I have it half done so I can finish it provide the test class when requested